<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WorkingDirectoryTemplate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Working directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="workingDirectoryBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="browseWorkingDirectoryBtn">
        <property name="text">
         <string>&amp;Browse...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="setWorkingDirectoryBtn">
        <property name="text">
         <string>&amp;Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QFrame" name="itemFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Item name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="itemNameBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Item value:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="itemValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Item:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="newItemBtn">
             <property name="text">
              <string>&amp;New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteItemBtn">
             <property name="text">
              <string>&amp;Delete</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="duplicateItemBtn">
             <property name="text">
              <string>Dupli&amp;cate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Filter:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filterBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="filterBtn">
             <property name="text">
              <string>&amp;Filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resetFilterBtn">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="itemList"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
